### PR TITLE
Add `4 Digit osu!mania World Cup 2023` article

### PR DESCRIPTION
**Pull request by fajar13k**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

## Description

- Initial article for upcoming [4 Digit osu!mania World Cup 2023](https://osu.ppy.sh/community/forums/topics/1671710)

### CODE REVIEW

File: wiki/Tournaments/4DM/2023/en.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+## Ruleset
+
+### Registration
+
+1. In order to be eligible for this tournament:
+   1. Player must be within the osu!mania 4-key rank range of #1,000 and #9,999 both when they submit their registration form and until the end of the registration phase.
+   2. Player must not have violated the osu! community rules in the past 12 months.
+   3. Player must not exceed the global osu!mania rank of #250.
+   4. Player must not have been a part of a former winning 4 Digit MWC Team.
+   5. Player must not appear on the list linked [here](https://docs.google.com/spreadsheets/d/1udZFfvt3it7Hk_k4J89k-UMzY1gfELaBN_oxw53CUOA/edit#gid=757705500). (There may be possibility to appeal their placement on this list. More information on this is given in the [screening phase](#screening-phase) section)
```
Comment (TicClick) [inline wiki/Tournaments/4DM/2023/en.md:89]:
```suggestion
   5. Player must not appear on the [Skillbans list](https://docs.google.com/spreadsheets/d/1udZFfvt3it7Hk_k4J89k-UMzY1gfELaBN_oxw53CUOA/edit#gid=757705500). (There may be possibility to appeal their placement on this list. More information on this is given in the [screening phase](#screening-phase) section)
```

File: wiki/Tournaments/4DM/2023/en.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+   2. Join the [4DM Discord server](https://discord.gg/W2MQ647) (Failure to join may result in disqualification)
+
+### Screening phase
+
+1. As player will read in the [prizes](#prizes) section, 4DM2023 plans to award the players on top 3 finishing teams with a **unique profile badge**. In order to make this happen, this tournament must meet the requirement of sending osu! staff a list of every player who signs up for this tournament in order for them to be screened for past or current violations of the osu! community rules. If players would like to read more about the screening process and profile badges, it is available through following [article](/wiki/osu!tourney/Prizes#profile-badges). If in this screening, osu! staff finds that player have violated the osu! community rules in the past 12 months, then we as host sadly will have to deny their registration as eligible to participate in 4DM2023. All players found in violation after the screening will be given time to appeal their stance, and if osu! staff changes their mind on a player's eligibility and notifies a host of this, then they will still be eligible for participation in 4DM2023!
```
Comment (TicClick) [inline wiki/Tournaments/4DM/2023/en.md:96]: (I added "the" somewhere but diff won't show it)
```suggestion
1. As player will read in the [prizes](#prizes) section, 4DM2023 plans to award the players on top 3 finishing teams with a **unique profile badge**. In order to make this happen, this tournament must meet the requirement of sending osu! staff a list of every player who signs up for this tournament in order for them to be screened for past or current violations of the osu! community rules. If players would like to read more about the screening process and profile badges, it is available through the following [article](/wiki/osu!tourney/Prizes#profile-badges). If in this screening, osu! staff finds that player have violated the osu! community rules in the past 12 months, then we as host sadly will have to deny their registration as eligible to participate in 4DM2023. All players found in violation after the screening will be given time to appeal their stance, and if osu! staff changes their mind on a player's eligibility and notifies a host of this, then they will still be eligible for participation in 4DM2023!
```

File: wiki/Tournaments/4DM/2023/en.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+2. Furthermore, there will be a **Skill Based Restriction System** developed and maintained by ::{ flag=TH }:: [HowToPlayLN](https://osu.ppy.sh/users/10879600), ::{ flag=FR }:: [Paturages](https://osu.ppy.sh/users/1375479), and ::{ flag=SG }:: [Polytetral](https://osu.ppy.sh/users/8612061) in order to keep 4DM2023 fair to all players and as free of malicious sandbagging as possible. To read more about the general scope of this system, please visit following [article](https://4digitmwc.github.io/skillban-articles/). To see who has been shown to be too skilled for the tournament by this system, please visit following [spreadsheet](https://docs.google.com/spreadsheets/d/1udZFfvt3it7Hk_k4J89k-UMzY1gfELaBN_oxw53CUOA/edit#gid=757705500).
```
Comment (TicClick) [inline wiki/Tournaments/4DM/2023/en.md:97]:
```suggestion
2. Furthermore, there will be a **skill-based restriction system** developed and maintained by ::{ flag=TH }:: [HowToPlayLN](https://osu.ppy.sh/users/10879600), ::{ flag=FR }:: [Paturages](https://osu.ppy.sh/users/1375479), and ::{ flag=SG }:: [Polytetral](https://osu.ppy.sh/users/8612061) in order to keep 4DM2023 fair to all players and as free of malicious sandbagging as possible. To read more about the general scope of this system, please visit the following [article](https://4digitmwc.github.io/skillban-articles/). To see who has been shown to be too skilled for the tournament by this system, visit the [Skillbans list](https://docs.google.com/spreadsheets/d/1udZFfvt3it7Hk_k4J89k-UMzY1gfELaBN_oxw53CUOA/edit#gid=757705500).
```

File: wiki/Tournaments/4DM/2023/en.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+16. If team scores are an exact tie at the conclusion of a map, the map will be replayed.
+17. If a team cannot gather 3 members into the match lobby by 10 minutes after the match’s scheduled start time, the opposing team will be declared a win by default. If neither team can gather 3 members into the match lobby by 10 minutes after the match’s scheduled start time, the team with the higher seed out of qualifiers will be declared a win by default.
+18. If a player disconnects in the middle of a match and the team cannot provide a replacement, the match can be delayed by up to 10 minutes before a win by default is declared to the opposing team.
+19. For additional rules left out of this section of the forum, please see the [Additional Rules](#additional-rules) section below. As well, feel free to ask a host about any questions regarding rules or the tournament in general!
+
+### Additional rules
+
+NOTE: Staff will attempt to closely follow the rules of the most recent 4K osu!mania World Cup whose wiki page is hyperlinked [here](/wiki/Tournaments/MWC/2022_4K) if you would like to read it.
```
Comment (TicClick) [inline wiki/Tournaments/4DM/2023/en.md:165]:
```suggestion
NOTE: Staff will attempt to closely follow the rules of [the most recent 4K osu!mania World Cup](/wiki/Tournaments/MWC/2022_4K).
```

**Review by TicClick — CHANGES_REQUESTED** [review]
in addition to that, I have to admit a lot of rules are VERY verbose here -- my gut feeling is that they can be at least halved without loss of meaning. not sure if it's something that needs to be addressed — may require massive rewrite and tinkering with the wording while I understand you aren't the one who wrote the rules — but worth noting regardless.

**Review by TicClick — DISMISSED** [review]
passing to @Walavouchey for a second look

**Comment by fajar13k** [comment]
Bumping this again if review is coming or I might just rewrite it with my usual style

**Review by Walavouchey — CHANGES_REQUESTED** [review]
posting partial review to prevent delay, may finish it with a bit more time, but i'd appreciate if you'd take a look at the last part yourself.

File: wiki/Tournaments/4DM/2023/en.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+   5. ...not appear on the [skillbans list](https://docs.google.com/spreadsheets/d/1udZFfvt3it7Hk_k4J89k-UMzY1gfELaBN_oxw53CUOA/edit#gid=757705500). (There may be a possibility to appeal one's placement on this list. More information on this is given in the [screening phase](#screening-phase) section.)
+2. To register for this tournament:
+   1. Submit the [player registration form](https://forms.gle/hhTmPVZjvbuDx7QSA).
+   2. Join the [4DM Discord server](https://discord.gg/W2MQ647). (Failure to join may result in disqualification.)
+
+### Screening phase
+
+1. The planned *unique profile badge* prize depends on 4DM2023's eligibility for [official tournament support](https://osu.ppy.sh/wiki/en/Tournaments/Official_support). Players will be [screened](https://osu.ppy.sh/wiki/en/Tournaments/Official_support#tournament-screening) by osu! staff and players found to be ineligible for tournament play will be denied entry. Consult [accounts@ppy.sh](mailto:accounts@ppy.sh) for appeals; if a failed screening result is retracted and notified of to a host, the player will still be eligible for participation in the tournament.
```
Comment (aceticke) [inline wiki/Tournaments/4DM/2023/en.md:96]:
```suggestion
1. The planned *unique profile badge* prize depends on 4DM2023's eligibility for [official tournament support](/wiki/en/Tournaments/Official_support). Players will be [screened](/wiki/en/Tournaments/Official_support#tournament-screening) by osu! staff and players found to be ineligible for tournament play will be denied entry. Consult [accounts@ppy.sh](mailto:accounts@ppy.sh) for appeals; if a failed screening result is retracted and notified of to a host, the player will still be eligible for participation in the tournament.
```

File: wiki/Tournaments/4DM/2023/en.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+  - This list of players shown on the sheet is subject to change until it has been sent to osu! staff for review. Any player appearing on this list may not sign up for 4DM2023, but there will be an appeal system. Players shown as **true** on both the ranked and tournament columns may not appeal their restriction to sign. However if they are only **true** for one or the other, they may message log.\#7606 via Discord with a request to appeal their restriction, which will be reviewed by tournament staff. For questions, refer to 4DM's `#questions` channel in the Discord server.
```
Comment (aceticke) [inline wiki/Tournaments/4DM/2023/en.md:98]:
```suggestion

  - This list of players shown on the sheet is subject to change until it has been sent to osu! staff for review. Any player appearing on this list may not sign up for 4DM2023, but there will be an appeal system. Players shown as **true** on both the ranked and tournament columns may not appeal their restriction to sign. However if they are only **true** for one or the other, they may message log.\#7606 via Discord with a request to appeal their restriction, which will be reviewed by tournament staff. For questions, refer to 4DM's `#questions` channel in the Discord server.
```

**Review by aceticke — CHANGES_REQUESTED** [review]
Should fix the errors that came up in the checks.

File: wiki/Tournaments/4DM/2023/en.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+10. Matches will be played with `Team VS` mode on `ScoreV2`.
+11. The Red Team will occupy slots 1–3, and the Blue Team will occupy slots 4–6.
+12. Allowed mods in the Free Mod pool are No Fail, Hidden, Fade In, Flashlight, and Mirror. The tiebreaker map is also played under Free Mod conditions.
+13. Teams have a maximum of 3 minutes after the conclusion of the previous map to pick another map from the Free Mod pool to play (and are expected to pick in 2 minutes or less). If a team takes longer than 3 minutes, a random beatmap from the Free Mod pool will be picked for them by the match referee.
+14. If a player disconnects during a match, their score is counted as a failed score. However, if this disconnect happens before 30 seconds of the map is played, the referee can decide to let the teams rematch. If this rematch happens, the rosters on both teams must stay the same unless the disconnected player cannot return to the lobby, in which case both teams are allowed to switch rosters.
+15. Failed scores do not count toward a team's total score, however a score which fails and then revives will still count. (Use NoFail if you believe you are in jeopardy of failing a map!)
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2023/en.md:158]:
```suggestion
15. Failed scores do not count towards a team's total score, however a score which fails and then revives will still count. (Use No Fail if you believe you are in jeopardy of failing a map!)
```

File: wiki/Tournaments/4DM/2023/en.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+16. If team scores are an exact tie at the conclusion of a map, the map will be replayed.
+17. If a team cannot gather 3 members into the match lobby by 10 minutes after the match’s scheduled start time, the opposing team will be declared a win by default. If neither team can gather 3 members into the match lobby by 10 minutes after the match’s scheduled start time, the team with the higher seed out of qualifiers will be declared a win by default.
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2023/en.md:160]:
```suggestion
17. If a team cannot gather 3 members into the match lobby by 10 minutes after the match's scheduled start time, the opposing team will be declared to win by default. If neither team can gather 3 members into the match lobby by 10 minutes after the match's scheduled start time, the team with the higher seed out of qualifiers will be declared to win by default.
```

File: wiki/Tournaments/4DM/2023/en.md [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+18. If a player disconnects in the middle of a match and the team cannot provide a replacement, the match can be delayed by up to 10 minutes before a win by default is declared to the opposing team.
+19. For additional rules left out of this section of the forum, please see the [Additional Rules](#additional-rules) section below. As well, feel free to ask a host about any questions regarding rules or the tournament in general!
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2023/en.md:162]:
```suggestion
19. For additional rules left out of this section of the forum, please see the [Additional rules](#additional-rules) section below. As well, feel free to ask a host about any questions regarding rules or the tournament in general!
```

File: wiki/Tournaments/4DM/2023/en.md [hunk 10]
```diff
@@ -0,0 +1,180 @@
...
+
+### Additional rules
+
+NOTE: Staff will attempt to closely follow the rules of [the most recent 4K osu!mania World Cup](/wiki/Tournaments/MWC/2022_4K).
+
+1. The tournament hosts are responsible for scheduling all matches and informing players of when they are scheduled to play. If a team is unhappy with when they are scheduled by the hosts to play, they have the option to reschedule their match as late as the end of **Thursday of the week before the weekend** of their match. Instructions on how to do so will be provided in the Discord server.
+2. Please note that while the organizers will try to schedule all matches for the best possible time for both teams, timezone conflicts may force some matches to occur in the early mornings or late nights.
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2023/en.md:169]:
```suggestion
2. Please note that while the organisers will try to schedule all matches for the best possible time for both teams, timezone conflicts may force some matches to occur in the early mornings or late nights.
```

File: wiki/Tournaments/4DM/2023/en.md [hunk 11]
```diff
@@ -0,0 +1,180 @@
...
+3. If no referee is available to referee a match, the match will be postponed.
+4. If no streamers are available to broadcast the match, the match will still occur without being broadcasted.
+5. If no commentators are available to commentate the match but broadcasters are available to broadcast the match, the match will still be broadcasted but without commentary. Please note that organizers will try to broadcast every match possible.
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2023/en.md:172]:
```suggestion
5. If no commentators are available to commentate the match but broadcasters are available to broadcast the match, the match will still be broadcasted but without commentary. Please note that organisers will try to broadcast every match possible.
```

File: wiki/Tournaments/4DM/2023/en.md [hunk 12]
```diff
@@ -0,0 +1,180 @@
...
+6. Rescheduled matches will be prioritized under non-rescheduled matches in terms of being broadcasted.
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2023/en.md:173]:
```suggestion
6. Rescheduled matches will be prioritised under non-rescheduled matches in terms of being broadcasted.
```

File: wiki/Tournaments/4DM/2023/en.md [hunk 13]
```diff
@@ -0,0 +1,180 @@
...
+7. Each respective stage of the tournament is to be completed in a single weekend, using the same mappool for the entire weekend. Weekday matches are allowed if both team captains agree to them and they take place before the weekend of that scheduled stage. Mappools will be released roughly a week before the stage they are to be played on. The latest a map in a released mappool may be slightly updated (e.g. as to fixing a notable timing or snapping issue) is by **23:59 UTC of the Monday after the mappool is released**.
+8. If one player on a team has their account restricted while still participating in the tournament for reasons of believed cheating, multi accounting, or anything else, this player's entire team shall be disqualified from the tournament with no exceptions. Depending on the case, the team most recently defeated in a match by the newly disqualified team may be allowed to advance in this disqualified team's place. If this happens, the result of this former match will be voided.
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2023/en.md:175]:
```suggestion
8. If one player on a team has their account restricted while still participating in the tournament for reasons of believed cheating, multi-accounting, or anything else, this player's entire team shall be disqualified from the tournament with no exceptions. Depending on the case, the team most recently defeated in a match by the newly disqualified team may be allowed to advance in this disqualified team's place. If this happens, the result of this former match will be voided.
```

File: wiki/Tournaments/4DM/2023/en.md [hunk 14]
```diff
@@ -0,0 +1,180 @@
...
+9. Repeated and excessive toxic behavior toward any staff member, player, or spectator either in match multiplayer chatroom, the official discord server, or the official twitch channel chatroom is strictly prohibited and will lead to, in successive order with host’s discretion to skip certain steps:
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2023/en.md:176]:
```suggestion
9. Repeated and excessive toxic behavior toward any staff member, player, or spectator either in match multiplayer chatroom, the official discord server, or the official twitch channel chatroom is strictly prohibited and will lead to, in successive order with host's discretion to skip certain steps:
```

File: wiki/Tournaments/4DM/2023/en.md [hunk 15]
```diff
@@ -0,0 +1,180 @@
...
+   1. Casual, undocumented warning
+   2. Formal, documented warning
+   3. Timeout of player wherever necessary for a subjective amount of time (Players on timeout during tournament match will be disallowed to play during their match and must be swapped out for other players)
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2023/en.md:179]:
```suggestion
   3. Timeout of player wherever necessary for a subjective amount of time (disallowing them from playing in tournament matches)
```

File: wiki/Tournaments/4DM/2023/en.md [hunk 16]
```diff
@@ -0,0 +1,180 @@
...
+   4. Official ban from the tournament as well as a ban from the official Discord server and Twitch channel
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2023/en.md:180]:
```suggestion
   4. Ban from the tournament as well as a ban from the official Discord server and Twitch channel
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**